transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/EncodeTransformationPipelineBuilder.scala

```diff
@@ -103,4 +105,33 @@ object EncodeTransformationPipelineBuilder extends PipelineBuilder[Args] {
       submittedBy = donorInput.read[String]("submitted_by")
     )
   }
+
+  def transformAntibody(antibodyInput: Msg): Antibody = {
```

Comment:
I've started moving these methods into their own objects, could you do the same?